workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSSendSystem.cs

```diff
@@ -82,10 +81,10 @@ private struct ComponentReplicator
             public ComponentGroup ReplicationComponentGroup;
             public List<ComponentGroup> CommandReplicationGroups;
 
-            public void Execute(Connection connection)
+            public void Execute(SpatialOSSendSystem sendSystem, Connection connection)
```

Comment:
Drop the first argument and pass in `this` directly into `SendCommands`

Reply:
no